src/themes/basic/_layout.styl

```diff
@@ -79,6 +79,15 @@ li input[type='checkbox']
   vertical-align middle
 
 /* navbar */
+body.right-sidebar .app-nav
+  margin 25px 0 0 60px
+  left 0
+  right unset
+
+  & > ul
+    padding-inline-start: 0;
+    padding-inline-end: 40px; // Chrome's default value.
+
```

Comment:
@jhildenbiddle I aimed to make all the changes in here non-breaking for the end user.

You may have some opinion on how better to organize this change. Happy to hear if you do!

Reply:
The reason for `padding-inline-end: 40px;` thing is to flip it to the other side when the menu is on the left, otherwise it has a default style in Chrome's user-agent style of `padding-inline-start: 40px;`.

Maybe `padding-inline-start: 40px should be hard-coded for non`right-sidebar` mode, otherwise that one can vary from browser to browser (I don't know if they settled on the same values, haven't checked).